front/src/store/utils.js

```diff
@@ -1,35 +1,65 @@
 import axios from 'axios'
 
-axios.defaults.baseURL = 'http://localhost:3333/'
+axios.defaults.baseURL = 'http://172.31.255.106:3333/'
```

Comment:
Because, i'm using my physical device to test react native application, so, it need access the server that is running in my computer. So, if the host stays as localhost, the cellphone don't has route to server.